tracpro/orgs_ext/tasks.py

```diff
@@ -229,3 +230,57 @@ def wrap_logger(self, level, org, msg, *args, **kwargs):
         kwargs.setdefault('org', org.name)
         msg = "{task} for {org}: " + msg
         return super(OrgTask, self).wrap_logger(level, msg, *args, **kwargs)
+
+
+@task(ignore_result=True)
+def fetch_runs(org_id, since):
+    """
+    Fetch responses for the org with id=org_id, going back
+    to `since` (datetime).
+
+    Creates or updates Response objects for each run.
+    """
+    from tracpro.polls.models import Poll, Response  # Avoid circular imports
+
+    logger.debug("Running fetch_runs")
```

Comment:
Well, it's debug rather than, say, info, so it won't clutter up the logs normally. Why log this at all, though... it's not really logging enough to debug anything anyway.  Maybe I'll just delete it.